cedar-policy-validator/src/cedar_schema/test.rs

```diff
@@ -528,7 +528,7 @@ namespace Baz {action "Foo" appliesTo {
             Extensions::all_available(),
         )
         .expect("Schema should parse");
-        assert!(warnings.collect::<Vec<_>>().is_empty());
+        assert!(warnings.next().is_none());
```

Comment:
maybe `assert_eq!(warnings.next(), None);` would give a better error message if this fails?  (actually displaying the warning in question?)

Reply:
may as well keep the collect and assert equal `vec![]` so it prints everything

Reply:
ah well, the warning type doesn't derive `Eq` and it's part of the public API, so I'll technically be making this change need a minor version release